src/components/Activity/activity-types.ts

```diff
@@ -1,7 +1,14 @@
-import actions from '../../actions/garden-action-types'
+import { GardenActionTypes as actions } from '@/actions/garden-action-types'
```

Comment:
Maybe we can do this later (not in this PR):

instead of maintaining types separately, we can have a global types declaration file which we can add it in the tsconfig.

This will keep all the global type declarations in one single place and it will make it easier for reference purposes at a later time as well.

Reply:
Agree :)